xc7/utils/prjxray_create_edges.py

```diff
@@ -463,14 +460,14 @@ def make_connection(conn, input_only_nodes, output_only_nodes,
         output_only_nodes (set of node_pkey): Nodes that can only be used as
             sources. This is because a synthetic tile will use this node as a
             sink.
-        find_wire (function): Return value from create_find_wire.
-        find_pip (function): Return value from create_find_pip.
-        find_connector (function): Return value from create_find_connector.
-        tile_name (str): Name of tile pip belongs too.
-        loc (prjxray.grid_types.GridLoc): Location of tile.
-        pip (prjxray.tile.Pip): Pip being connected.
+        tile_pkey (int): pkey of the tile in the tile table
+        src_wire_pkey (int): pkey of the source wire in the wire table
+        dst_wire_pkey (int): pkey of the destination wire in the wire table
+        loc (tuple): Location of tile.
```

Comment:
Whenever we pass the location of a tile, lets be specific about physical or VPR coordinates for clarity.

Reply:
You mean changing the table name from `tile` to eg. `vpr_tile` ?

Reply:
No, I just mean be specific when passing a location as a function argument.

Reply:
Ok.